_posts/blog/2019-05-20-Ruslan-GSoC.md

```diff
@@ -0,0 +1,78 @@
+excerpt: A new member of the Whitaker lab
+image:
+  feature:
+link:
+date: 2019-05-20
+modified:
+share: true
+author: ruslan_yermakov
+---
+
+## TL;DR 
+
+> My name is Ruslan Yermakov, I am a senior undergraduate student with a major in Computer Science at Sumy State University, Ukraine.
+I find it so hard to put into words how much I am enthusiastic to be accepted as a Google Summer of Code student 
+and excited to become a member of the Whitaker lab research group and to contribute to the scona project, an open-source toolkit for neuroscientists!
+My interests include data visualization, data science, neuroscience and design of intelligent algorithms and systems.
+
+## Why scona and neuroscience?
+
+Being initiative and enterprising, I am always willing to learn new things and look for new ideas. I value self-development and consider education to be the best investment. 
+Based on the Coursera’s recommendation once I’ve decided to complete the online course “Learning how to learn” (the best ever course) presented by Dr. Barbara Oakley. 
+This is how I have got excited about neuroscience field. What I liked the most, all the tips and suggestions of how to “learn smart not hard” were backed up by insightful 
+facts and recent researches, that inform about the way our brain works, which is helpful for improving one's productivity. 
+That’s why the chance to apply my knowledge and skills to develop the tool for neuroscientist was so appealing to me.
+
+From the list of INCF’s projects, Scona project grabbed my attention the most. It matches my previous experience and skills as well as my interests. 
+I especially appreciated the chance to contribute to the python project (love Python) that has great potential to be used by many people. 
+In short, I just felt that this project was right for me.
+
+## Some more interests
+
+In order to gain practical experience, in March 2018 I moved to Germany to work at SAP SE as a Software Engineer Intern for the following 6 months. 
+During my internship, I was working on the integration of Google Service - BigQuery into SAP DataHub (BigData platform). To complete the task I developed new components 
+(operators) in Python, covered code with unit tests and made well-written documentation. While working with teammates and mentors on the project, I became familiar with 
+the collaborative workflow and agile software development.
+
+I am a happy dog owner, football-addicted (once red, forever red), constantly broadening my outlook by engaging in exciting discussions about the meaning of life and 
+if a hotdog is a sandwich. I am interested in machine learning and artificial intelligence, new advancements and achievements in hightech. Love reading and hearing 
+awesome stories and points of views from interesting people (Elon Musk is not the only one). 
+
+<figure>
+  <img src="/images/Ruslan-GSoC/buddy.jpg"
+       alt="Buddy">
+  <figcaption> Waiting for the summer with my buddy </figcaption>
+</figure>
+
```

Comment:
Fantastic! I love the picture and this section in general. So lovely to get to know you!